src/navigation/BottomTabNavigator.tsx

```diff
@@ -0,0 +1,93 @@
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs'
+import React from 'react'
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import { useTheme } from 'styled-components/native'
+
+import {
+  BookOutlineIconGrey,
+  BookOutlineIconWhite,
+  HeartOutlineIconGrey,
+  HeartOutlineIconWhite,
+  HomeOutlineIconGrey,
+  HomeOutlineIconWhite,
+  StarOutlineIconGrey,
+  StarOutlineIconWhite
+} from '../../assets/images'
+import labels from '../constants/labels.json'
+import HomeStackNavigator from './HomeStackNavigator'
+import { RoutesParams } from './NavigationTypes'
+
+const BottomTabNavigator = createBottomTabNavigator<RoutesParams>()
+
+// TODO LUN-132, LUN-207, LUN-308: Remove this and use actual components
+const MockComponent = () => <></>
+
+const Navigator = (): JSX.Element | null => {
+  const theme = useTheme()
+
+  return (
```

Comment:
you should wrap the component with a  `<SafeAreaView style={{flex: 1}}>` to avoid overlapping bottom sheet on ios devices